internal/pkg/cli/app_status.go

```diff
@@ -49,18 +49,18 @@ func newAppStatusOpts(vars appStatusVars) (*appStatusOpts, error) {
 		appStatusVars: vars,
 		store:         ssmStore,
 		w:             log.OutputWriter,
-		initAppDescriber: func(o *appStatusOpts, envName, appName string) error {
-			d, err := describe.NewServiceDescriber(o.AppName(), envName, appName)
+		initAppDescriber: func(o *appStatusOpts, envName, svcName string) error {
+			d, err := describe.NewServiceDescriber(o.AppName(), envName, svcName)
 			if err != nil {
 				return fmt.Errorf("creating stack describer for project %s: %w", o.AppName(), err)
```

Comment:
the error message needs to be updated i think:
"create describer for service %s in environment %s and application %s"